Makefile

```diff
@@ -17,3 +17,16 @@ set-dev:
 test:
 	@echo "Running tests..."
 	export PYTHONPATH=${PWD}/src && python -m pytest utests/
+
+.PHONY: coverage
+coverage:
+	@echo "Running tests with coverage..."
+	export PYTHONPATH=${PWD}/src && python -m coverage run --source src/ -m pytest utests/
+
+.PHONY: coverage-check
+coverage-check:
+	@echo "Checking coverage..."
+	python -m coverage report --fail-under=28
```

Comment:
I know that 28 is the current coverage level, but as it is, it seems to be coming out of nowhere. I believe all code coverage tools have some mechanism to ensure that coverage for a PR hasn't decreased compared to the base branch. Is it possible to use this instead of setting an arbitrary target?

Reply:
We can certainly do that, but for now, it is enough to increase it from time to time.